app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -229,6 +232,119 @@ class RevisionCardFragmentTest {
     }
   }
 
+  @Test
+  fun testRevisionCard_extraLargeTextSize_hasCorrectDimension() {
+    launch<RevisionCardActivity>(
+      createRevisionCardActivityIntent(
+        context,
+        profileId.internalId,
+        FRACTIONS_TOPIC_ID,
+        subtopicId = 2,
+        FRACTIONS_SUBTOPIC_LIST_SIZE
+      )
+    ).use {
+      it.onActivity { activity ->
+        activity.revisionCardActivityPresenter
+          .loadRevisionCardFragment(ReadingTextSize.EXTRA_LARGE_TEXT_SIZE)
+      }
+      onView(withId(R.id.revision_card_explanation_text)).check(
+        matches(
+          withFontSize(
+            round(
+              context.resources.getDimension(
+                R.dimen.revision_card_fragment_font_scale_content_extra_large_text_size
+              )
+            )
+          )
+        )
+      )
+    }
+  }
```

Comment:
In our discussion we had agreed to hardcode the values, e.g below. 
```suggestion
        onView(withId(R.id.revision_card_explanation_text)).check(
        matches(
          withFontSize(
            62.0F
          )
        )
      )
```

I came up with 62F by logging the calculated value in the `FontSizeMatcher.kt` util class and checking logcat:
```
class FontSizeMatcher(private val expectedSize: Float) :
  TypeSafeMatcher<View?>(View::class.java) {
  override fun matchesSafely(target: View?): Boolean {
    if (target !is TextView) {
      return false
    }
    println("Textsize ${target.textSize}")
    return target.textSize.equals(expectedSize)
  }
```

You'll be able to find the appropriate values for other scale sizes too. Hope this helps.

Reply:
Thank you.